cloudmanager/resource_netapp_cloudmanager_connector_gcp.go

```diff
@@ -399,9 +400,7 @@ func getGCPServiceAccountKey(d *schema.ResourceData) (string, error) {
 		}
 		return string(serviceAccountKey), nil
 	} else if serviceAccountKey != "" {
-		return d.Get("service_account_key").(string), nil
-
+		return d.Get("serviceAccountKey").(string), nil
```

Comment:
can you double check this?

I think it should be either d.Get("service_account_key").(string)  or serviceAccountKey